### PR TITLE
feat: add woocommerce help page

### PR DESCRIPTION
**Pull request by kwiliwis**
This PR adds a help page for the Woocommerce plugin extension. It's draft until the plugin is published in WordPress marketplace. When it's done I'll add links and mark the PR as ready.

### CODE REVIEW

File: docs/integrations/wocommerce.md [hunk 1]
```diff
@@ -2,7 +2,92 @@
...
+    - Product feed name: Displays the feed title.
+    - Refresh interval: Choose between 24h, 12h, and 1h for feed data generation.
+    - Status: Indicates the current status of the feed.
+    - Products: Number of products in the feed.
+    - Progress: Displays the feed generation process percentage.
+    - Generated At: Date of the feed generation completion.
+    - Scheduled At: Next scheduled date for feed generation.
+    - Date: Date when the feed settings were created.
+
+2. Row Actions
```
[the remark this answers is not in the document]
Comment (andrejohansson) [reply docs/integrations/wocommerce.md:42]: https://daringfireball.net/projects/markdown/syntax#list

File: docs/integrations/wocommerce.md [hunk 2]
```diff
@@ -2,7 +2,92 @@
 sidebar_position: 1
 ---
 
-# WooCommerce
+# Woocommerce
+Prisjakt Feed Plugin User Guide
 
-We have a [WooCommerce](https://woocommerce.com/) plugin available for install upon request, please contact support@prisjakt.nu and we will help you.
+This guide provides detailed instructions on how to install and utilize the plugin to generate product feeds.
```
Comment (andrejohansson) [inline docs/integrations/wocommerce.md:8]: "utilize our WooCommerce plugin"?

we own it right?

File: docs/integrations/wocommerce.md [hunk 3]
```diff
@@ -2,7 +2,92 @@
...
 
+## Installation
+1. Installation via WordPress Marketplace:
+   - Navigate to your WordPress dashboard.
+   - Go to Plugins > Add New.
+   - Search for "Prisjakt Feed".
```
Comment (andrejohansson) [inline docs/integrations/wocommerce.md:14]: Small detail but perhaps backticks looks nicer thant quotes. What do you think?

compare: "this" to `this`

File: docs/integrations/wocommerce.md (hunk 2 above)
Comment (kwiliwis) [reply docs/integrations/wocommerce.md:8]: Short answer yes but there are some tiny details here. I updated this sentence.

File: docs/integrations/wocommerce.md [hunk 4]
```diff
@@ -2,7 +2,92 @@
...
+
+## Settings
+1. Plugin settings
+   - Customize global settings of the plugin.
+2. Extra fields
+   - Introduce additional fields in the product edit view, like prisjakt_id. This works for both simple and variable products.
+3. Plugin System Check
+   - Review system requirements and make any necessary improvements to your server as per the suggestions.
+
+## Final Notes
```
[the remark this answers is not in the document]
Comment (kwiliwis) [reply docs/integrations/wocommerce.md]: Renamed this section and added short info

File: docs/integrations/wocommerce.md [hunk 5]
```diff
@@ -2,7 +2,92 @@
...
+Actions
+
+    Add Filter: Introduce new conditions for filtering products.
+
+4. Category Mapping
+
+Map your product categories with Google categories for better alignment.
+
+## Settings
+1. Plugin settings
```
[the remark this answers is not in the document]
Comment (kwiliwis) [reply docs/integrations/wocommerce.md:91]: Thx, fixed

File: docs/integrations/wocommerce.md [hunk 6]
```diff
@@ -2,7 +2,92 @@
...
+
+### Adding New Feed
+1. Generating Feed
+
+   - Name: Define a name for the feed.
+   - Refresh interval: Schedule feed generation for every 24h, 12h, or 1h.
+
+2. Feed Mapping
+
+Map your product attributes to the Prisjakt shopping attributes.
```
[the remark this answers is not in the document]
Comment (kwiliwis) [reply docs/integrations/wocommerce.md]: Fixed

File: docs/integrations/wocommerce.md (hunk 3 above)
Comment (kwiliwis) [reply docs/integrations/wocommerce.md]: Fixed

File: docs/integrations/wocommerce.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kwiliwis) [reply docs/integrations/wocommerce.md]: Fixed

**Pull request merged**